### PR TITLE
Add pre-release option

### PR DESCRIPTION
**Pull request by mandarjog**

### CODE REVIEW

File: perf/istio-install/setup_istio.sh [hunk 1]
```diff
@@ -50,7 +50,7 @@ function install_istio() {
         if [[ ! -z "${HELMREPO_URL}" ]];then
           helmrepo="${HELMREPO_URL}"
         fi
-        helm repo add istio.io "${helmrepo}"
```
Comment (howardjohn) [inline perf/istio-install/setup_istio.sh]: we can just remove this whole helm block, we do all this stuff and then never use it

Comment (mandarjog) [reply perf/istio-install/setup_istio.sh]: ok good point.

Comment (mandarjog) [reply perf/istio-install/setup_istio.sh]: Remove section.

Comment (utka) [reply perf/istio-install/setup_istio.sh]: Does not work without this change, fails with this:
kubectl apply -f /home/prow/go/src/istio.io/tools/perf/istio-install/tmp/master-20190424-09-16/istio-init/files/
error: the path "/home/prow/go/src/istio.io/tools/perf/istio-install/tmp/master-20190424-09-16/istio-init/files/" does not exist

Comment (howardjohn) [reply perf/istio-install/setup_istio.sh]: Fixed in https://github.com/istio/tools/pull/111 @utka

Comment (utka) [reply perf/istio-install/setup_istio.sh]: Thank you!

**Pull request merged**